pipeline/extract.py

```diff
@@ -0,0 +1,200 @@
+'''File used for extracting information from bandcamp's website and API'''
```

Comment:
Docstrings need full stops.

```suggestion
'''File used for extracting information from bandcamp's website and API.'''
```

Reply:
I don't believe in full stops.